stable/anchore-admission-controller/values.yaml

```diff
@@ -103,6 +107,9 @@ credentials: {}
 
 # Settings related to init-ca pod
 initCa:
+  registry: ""
```

Comment:
Nit: To help prevent confusion for folks who aren't inclined to dig deep into the templates, should we populate initCa.registry=docker.io in the values file rather than using the default function in the template? alternatively, we could keep the current convention of having a flat `image` key.

Reply:
sure, will change that